public/js/components/util/Arrow.js

```diff
@@ -2,8 +2,9 @@
 
 const { DOM: dom } = require("react");
 
-require("./Arrow.css");
-
+if (typeof window == "object") {
```

Comment:
I don't like having to guard against this every single time we require a CSS file. I'd rather just do this:

```js
require.extensions['.css'] = function() {
  return {};
}
```

where the test runner initializes node. This will make requiring the CSS file a noop.